style/emoji/LICENSE

```diff
@@ -0,0 +1,3 @@
+Copyright 2014 Twitter, Inc and other contributors
```

Comment:
(This only applies to the images in the sprites, as the sprites were created by me)